packages/cli/src/deploy/resolveConfig.ts

```diff
@@ -22,7 +24,20 @@ export async function resolveConfig({
   readonly systems: readonly System[];
   readonly libraries: readonly Library[];
 }> {
-  const libraries = findLibraries(forgeOutDir).map((library): Library => {
+  const requirePath = await findUp("package.json", { cwd: process.cwd() });
```

Comment:
```suggestion
  const requirePath = await findUp("package.json");
```
I believe the `cwd` default is `process.cwd()`